tests/unit/Mail/MailerFactoryTest.php

```diff
@@ -1,43 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Model\Mail;
-
-use Codeception\Test\Unit;
-use Mockery as m;
-use Model\Payment\MailCredentials;
-use Nette\Mail\IMailer;
-use Nette\Mail\SmtpMailer;
-
-class MailerFactoryTest extends Unit
```

Comment:
Pro tohle nevidím test, ale vzhledem k tomu, že by měl být e2e test pro odesílání mailů, tak to nevidím jako takový problém.

Reply:
e2e asi tedka není, protoze není mailCatcher a nejaky mock OAuth

Reply:
To ale s mailCatcherem nijak nesouvisí. Tohle jen testuje ten přepínač.

Pokud je odesílání mailů vypnuto (dev mód) je vrácen testovací mailer. Pokud ne, tak je vrácen `OAuthMailer`.

Takže tohle nemá být dropnuto.

Reply:
Muzu to tam vrátit ale nedaří se mi najít, kde se to používá. Jako samostatný test mi to nejde spustit